modules/transactions/src/integrationTest/java/org/apache/ignite/internal/disaster/ItDisasterRecoveryManagerTest.java

```diff
@@ -95,6 +127,140 @@ void testRestartPartitions() {
         assertThat(selectAll(), hasSize(4));
     }
 
+    @Test
+    @ZoneParams(nodes = 2, replicas = 2, partitions = 2)
+    void testLocalPartitionStateTable() throws Exception {
```

Comment:
Please add

```
@WithSystemProperty(key = COLOCATION_FEATURE_FLAG, value = "true")
```

to all table-only tests so that, when we run these tests on a colocation-enabled branch, table tests don't suddenly fail